spotify_to_ytmusic/setup.py

```diff
@@ -43,6 +43,7 @@ def setup_spotify():
             "Paste your client id from the Spotify developer dashboard:"
         ),
         "client_secret": input("Paste your client secret from the Spotify developer dashboard:"),
+        "use_oauth": input("Use OAuth method for authorization (required for transfer liked songs) yes/no:"),
```

Comment:
This is very error prone. and allows invalid inputs.  I suggest separating the `input` statement, validating the response, and storing `True`, `False` in the settings.ini instead.

Reply:
I've tried to validate input before this implementation but fall back to just saving user's input, because `Setting` don't allow to store boolean value in settings.ini and we have to check strings when retrieving config anyway. 

I guess it's not very error prone cause any gibberish input will be interpreted as "no" answer but will be glad to see a better solution :)